tests/languages/sas/keyword_feature.test

```diff
@@ -1,13 +1,15 @@
 data else format if
 input proc run then
+libname set output options
 
 ----------------------------------------------------
 
 [
 	["keyword", "data"], ["keyword", "else"], ["keyword", "format"], ["keyword", "if"],
-	["keyword", "input"], ["keyword", "proc run"], ["keyword", "then"]
+	["keyword", "input"], ["keyword", "proc run"], ["keyword", "then"], ["keyword", "libname],

```

Comment:
You missed a quote `"` there after `libname`. That's why the test is failing.

Reply:
Thanks for bearing with me; this is my first github edit.